src/main/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -685,7 +685,11 @@ static String spliceOrAppendString(final String destination, final String toInse
             return destination.substring(0, insertIdx) + toInsert + destination.substring(insertIdx);
         }
         else {
-            return destination + toInsert;
+            if (destination.length() > 0 && destination.charAt(destination.length()-1) == '*') {
+                return destination;
+            } else {
+                return destination + toInsert;
+            }
         }
     }
 
```

Comment:
I couldn't comment on the vcf files.
Could you delete all the unused contigs?  Those files are fairly large.